aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,285 @@
-.. include:: /../common/stub-topic.txt
+ASP.NET Core Internationalization: Localization and Globalization 
+==================================================================
 
-|stub-icon| Localization
-========================
+By `Rick Anderson`_ with assistance from `Damien Bowden <https://twitter.com/damien_bod>`_, `Bart Calixto <https://twitter.com/bartmax>`_, and `Nadeem Afana <https://twitter.com/NadeemAfana>`__ 
 
-.. include:: /../common/stub-notice.txt
+Creating a multilingual website with ASP.NET Core will allow your site to reach a wider audience. ASP.NET has :doc:`/fundamentals/request-features` that provide support for localizing into different languages and cultures.
 
-.. _issue: https://github.com/aspnet/Docs/issues/653
\ No newline at end of file
+This sample is based on the **Localization.StarterWeb** project on `GitHub <https://github.com/aspnet/entropy>`__.
```

Comment:
I think this article could really use some introductory content. Imagine that I've never written a localized/globalized app in my life and now my job requires me to write one. What are globalization and localization? How are they even different? High level how are they accomplished in .NET and in ASP.NET Core? You have some content later in the article that I think you move up as intro material. I think you could also link to a bunch of stuff on MSDN (no need to replicate it here).